.ddev/config.yaml

```diff
@@ -32,7 +32,7 @@ web_extra_exposed_ports:
 web_extra_daemons:
   # Run vite in a separate process
   - name: 'vite'
-    command: 'npm install && npm run dev'
+    command: 'npm install && npm run build && npm run dev'
```

Comment:
Ensures that icons are present on start.

Reply:
Ah... you know what. I got thrown off by the name. 

I thought it would only copy the `public` directory. But it seems like you can configure it to copy any directory? 

- https://github.com/davidwebca/vite-plugin-hmr-public-copy?tab=readme-ov-file#copy

I was hesitant to use the `public` directory for icons, because on some of our more complex sites, we occasionally need to reference icons from JS. 

I'll put in a ticket to look into this more deeply. 

![CleanShot 2024-10-03 at 09 44 55@2x](https://github.com/user-attachments/assets/d479e847-bbac-4659-8b51-3f61029da054)

Reply:
Added

- [#49](https://github.com/vigetlabs/craft-site-starter/issues/49)